api/v1/services/StatsService.js

```diff
@@ -0,0 +1,137 @@
+            })
+            .fetchAll()
+            .then(populateStats('ecosystems'));
+            queries.push(ecosystemsQuery);
+
+            // School  stats
+            var schoolQuery = Attendee.query(function(qb){
+                qb.select('school as name').count('school as count').from('attendees').groupBy('school');
+            })
+            .fetchAll()
+            .then(populateStats('school'));
+            queries.push(schoolQuery);
+
+            // Transportation preference stats
+            var transportationQuery = Attendee.query(function(qb){
+                qb.select('transportation as name').count('transportation as count').from('attendees').groupBy('transportation');
+            })
+            .fetchAll()
+            .then(populateStats('transportation'));
+            queries.push(transportationQuery);
+
+            // Dietary preference stats
+            var dietQuery = Attendee.query(function(qb){
+                qb.select('diet as name').count('diet as count').from('attendees').groupBy('diet');
+            })
+            .fetchAll()
+            .then(populateStats('diet'));
+            queries.push(dietQuery);
+
+            // Shirt size stats
+            var shirtSizeQuery = Attendee.query(function(qb){
+                qb.select('shirt_size as name').count('shirt_size as count').from('attendees').groupBy('shirt_size');
+            })
+            .fetchAll()
+            .then(populateStats('shirtSize'));
+            queries.push(shirtSizeQuery);
+
+            // Gender stats
+            var genderQuery = Attendee.query(function(qb){
+                qb.select('gender as name').count('gender as count').from('attendees').groupBy('gender');
+            })
+            .fetchAll()
+            .then(populateStats('gender'));
+            queries.push(genderQuery);
+
+            // Year stats
+            var graduationYearQuery = Attendee.query(function(qb){
+                qb.select('graduation_year as name').count('graduation_year as count').from('attendees').groupBy('graduation_year');
+            })
+            .fetchAll()
+            .then(populateStats('graduationYear'));
+            queries.push(graduationYearQuery);
+
+            // Major stats
+            var majorQuery = Attendee.query(function(qb){
+                qb.select('major as name').count('major as count').from('attendees').groupBy('major');
+            })
+            .fetchAll()
+            .then(populateStats('major'));
+            queries.push(majorQuery);
+
+            // Novice stats
+            var isNoviceQuery = Attendee.query(function(qb){
+                qb.select('is_novice as name').count('is_novice as count').from('attendees').groupBy('is_novice');
+            })
+            .fetchAll()
+            .then(populateStats('isNovice'));
+            queries.push(isNoviceQuery);
+
+            // Total attendee stats
+            var attendeeQuery = Attendee.query(function(qb){
```

Comment:
This might not be in the same form as the others, but I think it'd be great as a helper method.